nixos/services/varnish/default.nix

```diff
@@ -171,7 +171,6 @@ in
       inherit (cfg) enable extraCommandLine http_address;
 
       enableConfigCheck = false;
-      stateDir = "/run/varnishd";
```

Comment:
Follow-up to the nixpkgs change we did a while back. Does this need some further changes in fc-nixos?

Reply:
I don't like that this happens in here. Is this required to make a test green?